noc-services/src/main/java/org/egov/noc/web/model/idgen/IdRequest.java

```diff
@@ -0,0 +1,29 @@
+package org.egov.noc.web.model.idgen;
```

Comment:
There are so many commits. Please create 2 pull requests. One for `noc-services`, also if the noc-service is designed specifically for `bpa` then the name should be `bpa-noc-services`. One pull request for `bpa-services` for noc integration

Reply:
noc-services is common for all modules not specific to bpa. 
ill go ahead and create separate for PR's.